recipes/gmp/all/test_package/conanfile.py

```diff
@@ -30,6 +34,6 @@ def test(self):
         if can_run(self):
             bin_path = os.path.join(self.cpp.build.bindirs[0], "test_package")
             self.run(bin_path, env="conanrun")
-            if self.options["gmp"].enable_cxx:
+            if self.enable_cxx:
```

Comment:
I think the correct way is `self.dependencies["gmp"].options.enable_cxx`

Reply:
it doesn't work currently

Reply:
Since this recipe has a `test_v1_package` this test recipe should only be ran using the V2 pipeline right? If this is correct then there shouldn't be a problem.

> Hi @StellaSmith, Yes, it is for Conan 2.0. Unfortunately, Conan 1.x does not support this construct in the test() method and CCI insists on remaining backwards compatible with Conan 1.x. That's why I moved this line into the generate() method as both 1.x and 2.0 support it there,. Thanks

Reply:
Hi @StellaSmith,
Maybe CCI has changed recently, but from my observations it seemed that Conan 1,x was executing both the test_v1_package and the test_package test suites.